tekton/resources/release/github_release.yaml

```diff
@@ -74,16 +76,16 @@ spec:
         EOF
     - name: filter-data
       image: gcr.io/tekton-releases/dogfooding/hub
-      workingdir: $(resources.inputs.source.path)
+      workingdir: $(workspaces.source.path)
       script: |
         #!/usr/bin/env bash
         set -ex
 
         # Restore full git history
-        git fetch --unshallow
+        git fetch --unshallow || true
```

Comment:
If this fails we should fail.

Reply:
right, *kinda* :stuck_out_tongue: 
The *main* problem is that, if the workspace *was* fully fetch (by a previous task, …), this fails because it is already *unshallowed*. I didn't see any way to make it fail only if the fetch fail and not if the repo was already *unshallowed*.. :sweat_smile:

Reply:
My take on adding the `|| true` is that if the fetch fail for any other reason that the repo is already unshallowed, the following commands (`git log` or `hub …`) will fail :sweat_smile: